### ISSUE TITLE
Click and drag MULTI_STEP doesn't persist

### ISSUE THREAD

**Issue by cjhendrix** [trevorsandy/lpub3d#544]
### Subject
In previous versions, the user could click and drag the MULTI_STEP object and a line like 
`0 !LPUB MULTI_STEP PLACEMENT CENTER PAGE INSIDE OFFSET 0 0.1`
would be added to the mpd. With the latest release, the line is added very briefly (visible in the command editor) but then disappears and the position of the multistep reverts back to the default. 

Adding the line manually in the command editor works as expected.

### Environment
* Version of LPub3D
![image](https://user-images.githubusercontent.com/1654485/116220668-2d581600-a74d-11eb-8b21-20ec6ed03934.png)
* Operating system - Win10, x86_64

**Comment by trevorsandy** [comment]
Thank you for reporting this behaviour.

Can you specify which multi step object(s) you are attempting to drag ? 

Movable objects parented by the step_group object are callouts and rotate icon. However, you can 'resize' objects that display a grabber - e.g. drag the grabber to reshape the PLI. For multi step pages, objects are intended to be moved using the 'Change Placement' context menu action. 

Which version of LPub3D allowed you to drag objects owned by step group other than callout and rotate icon. Keep in mind inserted items such as text or images are owned by the page object not the step group.

Cheers,

**Comment by cjhendrix** [comment]
Sorry! I pasted in the wrong line above which made it confusing.  I've edited it. 

It is the multistep group that is not retaining the new position. 

The version I was using where it was working is 2.4.2 r19 build 2451.

Thanks for looking into it.

**Comment by trevorsandy** [comment]
I can't reproduce this behaviour. Perhaps its been resolved with one of my recent updates. Once I post the next build, let me know if the behaviour persists for you ?

![Screenshot - 28_04_2021 , 08_28_48](https://user-images.githubusercontent.com/13388970/116356998-44a20c80-a7fc-11eb-8061-9a47580cae7d.png)

![Screenshot - 28_04_2021 , 08_30_26](https://user-images.githubusercontent.com/13388970/116357033-4ff53800-a7fc-11eb-979b-228d7e040749.png)

Cheers,

**Comment by cjhendrix** [comment]
I downloaded today's release (2.4.2 r96 build 2528) and still had the behavior, but I did a bit more testing and have narrowed down the conditions where the behavior occurs. 

The behavior only occurs when trying to drag the multistep group when there is already a ` MULTI_STEP PLACEMENT` command **later** in the model (or submodel). 

In the 2 page BI file pasted below, there is already a ` MULTI_STEP PLACEMENT` command on the **2nd** page. If you try to drag the multistep group on page 1, the position is not retained.

However, if we add `LOCAL` to that command on page 2, dragging the multistep group on page 1 works as expected. So this is an easy workaround. I suggest the desired behavior would be to have the `MULTI_STEP PLACEMENT` command be added to the file regardless of what comes later in the file (and indeed it is added, for a split second, when the drag operation is completed). 

```
0 !LPUB MULTI_STEP BEGIN
1 15 -40 0 -80 1 0 0 0 1 0 0 0 1 3001.dat
0 STEP
1 1 -40 0 -40 1 0 0 0 1 0 0 0 1 3001.dat
0 STEP
0 !LPUB MULTI_STEP END
0 !LPUB MULTI_STEP BEGIN
0 !LPUB MULTI_STEP PLACEMENT CENTER PAGE INSIDE OFFSET -0.0173 0.0200
1 71 -40 0 0 1 0 0 0 1 0 0 0 1 3001.dat
0 STEP
1 72 -40 0 40 1 0 0 0 1 0 0 0 1 3001.dat
0 STEP
0 !LPUB MULTI_STEP END
```
